src/entrypoints/core.ts

```diff
@@ -1,5 +1,4 @@
 // Compat needs to be first import
-import "../resources/compatibility";
```

Comment:
This needs to be the first import, as per the comment…

Reply:
Stupid auto import organizer... :-/